news_items/040_manual_grading_ui/index.html

```diff
@@ -0,0 +1,92 @@
+<style>
+ img {
+     display: block;
+     margin-left: auto;
+     margin-right: auto;
+     margin-top: 15px;
+     margin-bottom: 15px;
+ }
+ pre {
+     width: 90%;
+     margin-left: auto;
+     margin-right: auto;
+     overflow: hidden;
+     margin-bottom: 1rem;
+     font-family: SFMono-Regular, Menlo, Monaco, Consolas, "Liberation Mono", "Courier New", monospace;
+     background: #f0f0f0;
+     padding: 1em;
+     padding-bottom: 0em;
+ }
+</style>
+
+<p>
+  A major overhaul of manual grading features has just been deployed
+  to all courses. Instructors and TAs now have more flexibility to
+  grade question components manually, and to more easily review and
+  provide feedback on students work.
+</p>
+<p>
+  Assessment questions can now be assigned a combination of
+  auto-graded and manual point components. This combination can be
+  used in questions that require some level of human review, such as
+  questions that require a description of the progress to arrive at
+  the result ("show your work"), review of code style and comments in
+  coding questions, or essay-type justification of student
+  answers. Assessments can be set up to use this combination in
+  individual questions:
+</p>
+<pre>
+"zones": [
+    {
+        "title": "Manually graded questions",
+        "questions": [
+            {"id": "essayQ", "manualPoints": 10},
+            {"id": "autoPlusManualQ", "autoPoints": 7, "manualPoints": 15},
+        ]
+    }
+</pre>
+<p>
+  Additional details on this configuration can be found
+  in <a href="https://prairielearn.readthedocs.io/en/latest/assessment/#question-specification"
+  target="_blank">the PrairieLearn documentation</a>.
+</p>
+<p>
+  Manual grading has also received a new user interface, in addition
+  to the existing grade upload features. It is now possible to
+  manually grade student submissions directly in the PrairieLearn user
+  interface, allowing course staff to provide feedback and individual
```

Comment:
Is this what was meant here?
```suggestion
  interface, allowing course staff to provide feedback on individual
```